OutOfSchool/OutOfSchool.WebApi/Services/ParentService.cs

```diff
@@ -122,6 +129,40 @@ public Task<ShortUserDto> Update(ShortUserDto dto)
         return ExecuteUpdate(dto);
     }
 
+    /// <inheritdoc/>
+    public async Task<Result<bool>> BlockParent(ParentBlockByAdminDto parentBlock)
+    {
+        logger.LogInformation("Changing Block status of Parent by ParentId started. Looking ParentId is {Id}", parentBlock.ParentId);
+        var parent = await repositoryParent.GetByIdWithDetails(parentBlock.ParentId, "User").ConfigureAwait(false);
+        if (parent is null)
+        {
+            return Result<bool>.Failed(new OperationError
+            {
+                Code = "404",
```

Comment:
(Code style) Beforementioned literals aren't good for your code maintainability. Use constants @DKymachynskyi !

Btw I'm not sure if you should return different codes for not found/already (un)blocked parents. Was it in initial task?Providing info about parentId absence/existence you give security leakage that is not somehow logged by `parentBlockedByAdminLogService.SaveChangesLogAsync`.